mmv1/third_party/cai2hcl/converter_test.go

```diff
@@ -0,0 +1,28 @@
+package cai2hcl
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v5/cai2hcl/services"
+	cai2hclTesting "github.com/GoogleCloudPlatform/terraform-google-conversion/v5/cai2hcl/testing"
+)
+
+func TestConvertCompute(t *testing.T) {
```

Comment:
Ideally these tests would live in the respective service packages - is that possible?

Reply:
This already works like that - each converter has its own test and uses local UberConverter to execute the conversion.

This test is a glue test to check the integration point between services